react/features/base/react/components/native/JitsiSectionList.js

```diff
@@ -0,0 +1,113 @@
+// @flow
+
+import React, { Component } from 'react';
+import {
+    SafeAreaView,
+    SectionList
+} from 'react-native';
+
+import styles from './styles';
+
+/**
+ * The type of the React {@code Component} props of {@link JitsiSectionList}
+ */
+type Props = {
+
+    /**
+     * Rendered when the list is empty. Can be a React Component Class, a render
+     * function, or a rendered element.
+     */
+    ListEmptyComponent: Object,
+
+    /**
+    *
+    * Used to extract a unique key for a given item at the specified index.
+     * Key is used for caching and as the react key to track item re-ordering.
+    */
+    keyExtractor: Function,
+
+    /**
+    *
+    * Functions that defines what happens when the list is pulled for refresh
+    */
+    onRefresh: Function,
+
+    /**
+    *
+    * A boolean that is set true while waiting for new data from a refresh.
+    */
+    refreshing: ?boolean,
+
+    /**
+    *
+    * Default renderer for every item in every section.
+    */
+    renderItem: Function,
+
+    /**
+    *
+    * A component rendered at the top of each section. These stick to the top
+     * of the ScrollView by default on iOS.
+    */
+    renderSectionHeader: Object,
+
+    /**
+     * Sections to be rendered in the following format:
+     *
+     * [
+     *   {
+     *     title: string,               <- section title
+     *     key: string,                 <- unique key for the section
+     *     data: [                      <- Array of items in the section
+     *       {
+     *         colorBase: string,       <- the color base of the avatar
+     *         title: string,           <- item title
+     *         url: string,             <- item url
+     *         lines: Array<string>     <- additional lines to be rendered
+     *       }
+     *     ]
+     *   }
+     * ]
+     */
+    sections: Array<Object>
+};
+
+/**
+ * Implements a React Native {@link Component} that wraps the React Native
+ * SectionList component in a SafeAreaView so that it renders the sectionlist
+ * within the safe area of the device
+ *
+ * @extends Component
+ */
+export default class JitsiSectionList extends Component<Props> {
```

Comment:
can we drop the "Jitsi" prefix? We don't use it on any UI components AFAICT.

Reply:
- [ ] drop jitsi prefix